portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,54 @@
   margin-left: auto;
```

Comment:
It might be nice to comment this file into sections so that it's easy to find/figure out what each of these CSS declarations are for